pages/gm/f21.js

```diff
@@ -84,16 +111,86 @@ function F21() {
 				}}
 			/>
 			<Banner decorative />
+      <div className="text-center">
+        {/* eslint-disable-next-line @next/next/no-img-element */}
+        <img className="gm-graphic" src="/images/Fall_GM_2021_graphic.png" width="1000px" alt="Fall GM 2021 Marketing Graphic"/>
+      </div>
+      <Countdown date={GM_START_TIME} renderer={countdownRenderer}/>
 			<div className="content-container-tight text-center">
-				<h1>Fall &apos;21 General Meeting</h1>
-        <Link href={RSVP_LINK}>
-          <a className="button" target="_blank" rel="noreferrer noopener">
-            RSVP Now!
-          </a>
-        </Link>
-        <p>Check back later for more information on COVID safety, location, and the agenda!!</p>
+        <div id="info-wrapper">
+          <h2>Relevant information</h2>
+          <div className="flex">
+            <div>
+              <h3>How to get there</h3>
+              {/* eslint-disable-next-line @next/next/no-img-element */}
+              <img src="/images/Fall_GM_2021_temp_map.png" alt="temporary FPO of google maps" width="400px"/>
+            </div>
+            <div className="what-to-bring">
+              <h3>What to bring</h3>
+              <p>Required: Face Mask, Bruincard</p>
```

Comment:
```suggestion
              <p>Required: Face Mask, Bruincard.</p>
```

(punctuation consistency w/ next paragraph)